### DISCUSSION TITLE
[Question][Hw6] KCLOUDVPN access issue

### DISCUSSION THREAD

**Discussion by ZosiaZamoyska** [prosyslab-classroom/cs424-program-reasoning#308] [Q&A]
Hello,

I am having an issue accessing Kcloud VPN, a following message pops out:
 
<img width="344" alt="Screen Shot 2022-12-08 at 2 25 30 PM" src="https://user-images.githubusercontent.com/19697296/206363884-a3d8fcf4-8337-4488-a0d1-2b85ff83f089.png">

I am using macOS. I am wondering if other students also have or had this issue. 
Also, what should I do about the homework if this issue continues and I cannot access my solution?

**Comment by Lee-Janggun** [comment]
Did you try allowing SecuwaySSL in the settings? It's been a while since I used it so my memory is a bit vague, but I think we can give it permission.

I also remember that the reason I stopped using it was because of such problems...
